src/main/app/src/components/toteutus/ToteutusInfoGrid.js

```diff
@@ -22,25 +23,33 @@ const useStyles = makeStyles((theme) => ({
 const suunniteltuKesto = (t, vuosi, kk) => {
   if (!vuosi && !kk) {
     return t('koulutus.ei-kestoa');
-  } else {
-    return [
-      vuosi === '1'
-        ? t('koulutus.kesto-vuosi')
-        : vuosi
-        ? t('koulutus.kesto-vuosia', { vuosi })
-        : null,
-      kk === '1'
-        ? t('koulutus.kesto-kuukausi')
-        : kk
-        ? t('koulutus.kesto-kuukautta', { kk })
-        : null,
-    ].join('\n');
   }
+
+  return [
+    vuosi === 1
+      ? t('koulutus.kesto-vuosi')
+      : vuosi
+      ? t('koulutus.kesto-vuosia', { vuosi })
+      : null,
+    kk === 1
+      ? t('koulutus.kesto-kuukausi')
+      : kk
+      ? t('koulutus.kesto-kuukautta', { kk })
+      : null,
```

Comment:
Tämäkin on jo ilmeisesti aiemmasta kommentista, mutta olisi lyhyempi ja selkeämpi käyttäen `i18next`:in monikko-ominaisuutta: https://www.i18next.com/translation-function/plurals